lint_test/analysis_options.yaml

```
@@ -6,23 +6,43 @@ custom_lint:
   rules:
     - cyclomatic_complexity:
       max_complexity: 4
+      exclude:
+        - method_name: excludeMethod
+          class_name: Exclude
+        - method_name: excludeMethod      
     - number_of_parameters:
       max_parameters: 2
+      exclude:
+        - method_name: avoidNumberOfParameters
+          class_name: Exclude
+        - method_name: avoidNumberOfParameters          
     - function_lines_of_code:
       max_lines: 50
     - avoid_non_null_assertion
     - avoid_late_keyword:
       allow_initialized: true
     - avoid_global_state
-    - avoid_returning_widgets
+    - avoid_returning_widgets:
+      exclude:
+        - method_name: excludeWidgetMethod
+          class_name: ExcludeWidget
```

Let's swap like this -- reads better IMO
```suggestion
        - class_name: ExcludeWidget
          method_name: excludeWidgetMethod
```